tests/test_utils.py

```diff
@@ -817,8 +820,15 @@ def test_generic(self) -> None:
         assert utils.resolve_annotation(annotation, globals(), locals(), {}) == list[int]
 
     # alias and arg in local scope
-    def test_forwardref_local(self) -> None:
-        IntOrStr = Union[int, str]
+    @pytest.mark.parametrize(
+        "typehint",
+        [
+            Union[int, str],  # noqa: UP007
+            int | str,
+        ],
+    )
+    def test_forwardref_local(self, typehint) -> None:
+        IntOrStr = typehint
```

Comment:
```suggestion
    def test_forwardref_local(self) -> None:
        IntOrStr = int | str
```
The point of this test isn't unions, it was more of a means to an end; we can simply use the new syntax here.